media-downloader/cmd/sync.go

```
@@ -0,0 +1,556 @@
+				}
+			},
+		})
+	}
+
+	estimatedDuration := time.Duration(float64(totalPayloads)/float64(constants.TMDB_CLIENT_RATE_LIMIT*len(proxies))) * time.Second
+	timeout := max(estimatedDuration+(30*time.Minute), 30*time.Second)
+
+	logger.I.Info("Waiting for downloads to complete", zap.Duration("estimatedTimeout", timeout))
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+	rm.WaitUntilIdle(ctx)
```

golangci-lint (1.64.8)

Error return value of `rm.WaitUntilIdle` is not checked

(errcheck)